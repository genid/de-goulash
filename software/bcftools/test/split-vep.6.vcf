##fileformat=VCFv4.2
##contig=<ID=1,length=248956422,assembly=GRCh38>
##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|SYMBOL_SOURCE|HGNC_ID|CANONICAL|MANE|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|GENE_PHENO|SIFT|PolyPhen|DOMAINS|miRNA|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|AA_AF|EA_AF|gnomAD_AF|gnomAD_AFR_AF|gnomAD_AMR_AF|gnomAD_ASJ_AF|gnomAD_EAS_AF|gnomAD_FIN_AF|gnomAD_NFE_AF|gnomAD_OTH_AF|gnomAD_SAS_AF|MAX_AF|MAX_AF_POPS|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|existing_InFrame_oORFs|existing_OutOfFrame_oORFs|existing_uORFs|five_prime_UTR_variant_annotation|five_prime_UTR_variant_consequence|SpliceRegion|LoF|LoF_filter|LoF_flags|LoF_info|SpliceAI_pred_DP_AG|SpliceAI_pred_DP_AL|SpliceAI_pred_DP_DG|SpliceAI_pred_DP_DL|SpliceAI_pred_DS_AG|SpliceAI_pred_DS_AL|SpliceAI_pred_DS_DG|SpliceAI_pred_DS_DL|SpliceAI_pred_SYMBOL">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	979496	.	T	C	.	.	CSQ=C|missense_variant|MODERATE|PERM1|ENSG00000187642|Transcript|ENST00000341290|protein_coding|3/5||ENST00000341290.6:c.1192A>G|ENSP00000343864.2:p.Ser398Gly|1228|1192|398|S/G|Agc/Ggc|rs13302983||-1||SNV|HGNC|HGNC:28208|||2|A2||ENSP00000343864|Q5SV97||||tolerated(0.2)|benign(0.003)|PANTHER:PTHR47282&MobiDB_lite:mobidb-lite||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000379407|protein_coding||||||||||rs13302983|4488|1||SNV|HGNC|HGNC:25284|||1|A2|CCDS53256.1|ENSP00000368717|Q494U1||||||||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000379409|protein_coding||||||||||rs13302983|4488|1||SNV|HGNC|HGNC:25284|||2|A2||ENSP00000368719|Q494U1||||||||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000379410|protein_coding||||||||||rs13302983|3631|1||SNV|HGNC|HGNC:25284|YES|NM_032129.3|1|P3|CCDS4.1|ENSP00000368720|Q494U1||UPI00001416D8||||||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1,C|missense_variant|MODERATE|PERM1|ENSG00000187642|Transcript|ENST00000433179|protein_coding|1/3||ENST00000433179.3:c.1534A>G|ENSP00000414022.3:p.Ser512Gly|1534|1534|512|S/G|Agc/Ggc|rs13302983||-1||SNV|HGNC|HGNC:28208|YES||5|P2|CCDS76083.1|ENSP00000414022|Q5SV97||||tolerated(0.18)|benign(0.011)|PANTHER:PTHR47282&MobiDB_lite:mobidb-lite||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1,C|upstream_gene_variant|MODIFIER|PERM1|ENSG00000187642|Transcript|ENST00000479361|retained_intron||||||||||rs13302983|2855|-1||SNV|HGNC|HGNC:28208|||1||||||||||||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000491024|protein_coding||||||||||rs13302983|3631|1|cds_start_NF|SNV|HGNC|HGNC:25284|||3|||ENSP00000462558||J3KSM5|UPI000268AE1F||||||||0.9418|0.9539|0.999|0.9592|0.9847|||0.9721|0.9533|0.9754|0.9166|0.9999|0.9923|0.9677|0.9536|0.9811|0.9999|gnomAD_EAS|||||||||||||||||||-27|-2|23|25|0.00|0.00|0.00|0.00|PERM1
1	979560	.	T	C	.	.	CSQ=C|synonymous_variant|LOW|PERM1|ENSG00000187642|Transcript|ENST00000341290|protein_coding|3/5||ENST00000341290.6:c.1128A>G|ENSP00000343864.2:p.Ala376%3D|1164|1128|376|A|gcA/gcG|rs13303033||-1||SNV|HGNC|HGNC:28208|||2|A2||ENSP00000343864|Q5SV97||||||PANTHER:PTHR47282&Low_complexity_(Seg):seg|||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000379407|protein_coding||||||||||rs13303033|4552|1||SNV|HGNC|HGNC:25284|||1|A2|CCDS53256.1|ENSP00000368717|Q494U1|||||||||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000379409|protein_coding||||||||||rs13303033|4552|1||SNV|HGNC|HGNC:25284|||2|A2||ENSP00000368719|Q494U1|||||||||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000379410|protein_coding||||||||||rs13303033|3695|1||SNV|HGNC|HGNC:25284|YES|NM_032129.3|1|P3|CCDS4.1|ENSP00000368720|Q494U1||UPI00001416D8|||||||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1,C|synonymous_variant|LOW|PERM1|ENSG00000187642|Transcript|ENST00000433179|protein_coding|1/3||ENST00000433179.3:c.1470A>G|ENSP00000414022.3:p.Ala490%3D|1470|1470|490|A|gcA/gcG|rs13303033||-1||SNV|HGNC|HGNC:28208|YES||5|P2|CCDS76083.1|ENSP00000414022|Q5SV97||||||PANTHER:PTHR47282&Low_complexity_(Seg):seg|||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1,C|upstream_gene_variant|MODIFIER|PERM1|ENSG00000187642|Transcript|ENST00000479361|retained_intron||||||||||rs13303033|2919|-1||SNV|HGNC|HGNC:28208|||1|||||||||||||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1,C|downstream_gene_variant|MODIFIER|PLEKHN1|ENSG00000187583|Transcript|ENST00000491024|protein_coding||||||||||rs13303033|3695|1|cds_start_NF|SNV|HGNC|HGNC:25284|||3|||ENSP00000462558||J3KSM5|UPI000268AE1F|||||||0.5096|0.177|0.6095|0.7123|0.5895|0.5971|||0.5887|0.2428|0.6604|0.5617|0.7237|0.5334|0.5801|0.5647|0.6213|0.7237|gnomAD_EAS|||||||||||||||||||28|-30|-39|3|0.00|0.00|0.00|0.00|PERM1
1	999842	.	C	A	.	.	CSQ=A|intron_variant|MODIFIER|HES4|ENSG00000188290|Transcript|ENST00000304952|protein_coding||1/3|ENST00000304952.11:c.108+24G>T|||||||rs2298214||-1||SNV|HGNC|HGNC:24149||NM_021170.4|1|P1|CCDS5.1|ENSP00000304595|Q9HCC6|||||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|missense_variant|MODERATE|HES4|ENSG00000188290|Transcript|ENST00000428771|protein_coding|1/3||ENST00000428771.6:c.132G>T|ENSP00000393198.2:p.Arg44Ser|331|132|44|R/S|agG/agT|rs2298214||-1||SNV|HGNC|HGNC:24149|YES||2||CCDS44034.1|ENSP00000393198||E9PB28|UPI0001881B51||tolerated_low_confidence(0.72)|benign(0)|MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg|||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS||||||||||||||||||MUTANT_DONOR_MES:11.0784944978449&EXON_END:1000172&INTRON_END:999691&DE_NOVO_DONOR_MES_POS:-156&DE_NOVO_DONOR_POS:-156&EXON_START:999692&DE_NOVO_DONOR_MES:7.0006176073036&DE_NOVO_DONOR_PROB:0.167156964545091&INTRON_START:999614|6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|non_coding_transcript_exon_variant|MODIFIER|HES4|ENSG00000188290|Transcript|ENST00000481869|retained_intron|1/2||ENST00000481869.1:n.255G>T||255|||||rs2298214||-1||SNV|HGNC|HGNC:24149|||2|||||||||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|intron_variant|MODIFIER|HES4|ENSG00000188290|Transcript|ENST00000484667|protein_coding||1/2|ENST00000484667.2:c.108+24G>T|||||||rs2298214||-1||SNV|HGNC|HGNC:24149|||3|||ENSP00000425085||D6REB3|UPI0001D3BBEE|||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|upstream_gene_variant|MODIFIER|AL645608.7|ENSG00000272512|Transcript|ENST00000606034|lncRNA||||||||||rs2298214|1791|-1||SNV|Clone_based_ensembl_gene||YES|||||||||||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|upstream_gene_variant|MODIFIER|ISG15|ENSG00000187608|Transcript|ENST00000624652|protein_coding||||||||||rs2298214|1303|1|cds_end_NF|SNV|HGNC|HGNC:4053|||3|||ENSP00000485313||A0A096LNZ9||1||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|upstream_gene_variant|MODIFIER|ISG15|ENSG00000187608|Transcript|ENST00000624697|protein_coding||||||||||rs2298214|1296|1||SNV|HGNC|HGNC:4053|||3|A2||ENSP00000485643||A0A096LPJ4||1||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||6|-37|6|24|0.00|0.00|0.01|0.02|HES4,A|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000000118|promoter||||||||||rs2298214||||SNV||||||||||||||||||0.4938|0.0356|0.6268|0.7718|0.5895|0.6339|0.1386|0.5695|0.6136|0.1208|0.7139|0.6344|0.786|0.5555|0.595|0.5882|0.6625|0.786|gnomAD_EAS|||||||||||||||||||||||||||
1	12716215	.	C	A	.	.	CSQ=A|stop_gained|HIGH|AADACL3|ENSG00000188984|Transcript|ENST00000359318|protein_coding|1/4||ENST00000359318.8:c.39C>A|ENSP00000352268.6:p.Cys13Ter|106|39|13|C/*|tgC/tgA|rs3000860||1||SNV|HGNC|HGNC:32037|YES|NM_001103170.3|3|P1|CCDS41253.2|ENSP00000352268|Q5VUY0||UPI0003EAEC70||||PIRSF:PIRSF037251&PANTHER:PTHR23024&PANTHER:PTHR23024:SF115&Transmembrane_helices:TMhelix|||0.1436|0.1188|0.1052|0.2679|0.1362|0.0838||||||||||||0.2679|EAS|||||||||||||||HC|||GERP_DIST:92.57575&BP_DIST:1182&PERCENTILE:0.0318627450980392&DIST_FROM_LAST_EXON:408&50_BP_RULE:PASS&PHYLOCSF_TOO_SHORT|||||||||,A|non_coding_transcript_exon_variant|MODIFIER|AADACL3|ENSG00000188984|Transcript|ENST00000620146|processed_transcript|1/3||ENST00000620146.2:n.101C>A||101|||||rs3000860||1||SNV|HGNC|HGNC:32037|||2|||||||||||||0.1436|0.1188|0.1052|0.2679|0.1362|0.0838||||||||||||0.2679|EAS|||||||||||||||||||||||||||,A|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000347677|promoter_flanking_region||||||||||rs3000860||||SNV||||||||||||||||||0.1436|0.1188|0.1052|0.2679|0.1362|0.0838||||||||||||0.2679|EAS|||||||||||||||||||||||||||
