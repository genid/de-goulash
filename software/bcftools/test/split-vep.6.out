14464	1
14469	0.5
14522	0.2
